<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectImportingOptions</class>
 <widget class="QDialog" name="SelectImportingOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <property name="whatsThis">
   <string/>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QComboBox" name="scanType">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>20</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <item>
    <property name="text">
     <string>Flair</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>T1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>T2</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="scanTypeInstructions">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>251</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>If known, accuracy of segmentations can be improved greatly ()</string>
   </property>
   <property name="text">
    <string>Select scan type:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="segTypeInstructions">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>421</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>Parts of the MRI scan to be viewed</string>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
   <property name="text">
    <string>Select the desired segmentations</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="brainSeg">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>100</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Brain Segmentation</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="tumorSeg">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>150</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Tumor Segmentation</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="modelSelectionsInstructions">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>230</y>
     <width>571</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>Models to use during processing (axial model strongly recommended)</string>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
   <property name="text">
    <string>Select the desired models to use during segmentation</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="axialModel">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>240</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Axial Model</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="corticalModel">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>290</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Cortical Model</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="sagittalModel">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>340</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Saggital Model</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="modelSelectionsInstructions2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>290</y>
     <width>571</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>The more models, the higher the precision, but also the longer processing will take</string>
   </property>
   <property name="text">
    <string>(More models will produce a result with more </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="modelSelectionsInstructions3">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>320</y>
     <width>571</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>The more models, the higher the precision, but also the longer processing will take</string>
   </property>
   <property name="text">
    <string>precision, but will also take longer to process)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="confirmButton">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>700</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Confirm</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>700</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="images">
   <property name="geometry">
    <rect>
     <x>-30</x>
     <y>700</y>
     <width>511</width>
     <height>41</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Create a folder in the destination that stores all the segmentations</string>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="text">
    <string>Write out segmentations as images</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>440</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="modelSelectionsInstructions2_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>430</y>
     <width>571</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>The more models, the higher the precision, but also the longer processing will take</string>
   </property>
   <property name="text">
    <string>Please Enter patient age:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="modelSelectionsInstructions2_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>550</y>
     <width>571</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>The more models, the higher the precision, but also the longer processing will take</string>
   </property>
   <property name="text">
    <string>Please enter time interval between scans</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_2">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>560</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
